src/components/pages/Home/RenderHomePage.js

```diff
@@ -85,8 +85,8 @@ function RenderHomePage(props) {
   const user = useContext(UserContext);
 
   useEffect(() => {
-    axios
-      .get(`${process.env.REACT_APP_API_URI}/cases`, {
+    axiosWithAuth()
+      .get(`/cases`, {
         headers: {
```

Comment:
axiosWithAuth takes care of headers and authorization so these should be removed

Reply:
Good catch! I will go in and remove the headers and push a new commit.